proto/tendermint/abci/types.proto

```diff
@@ -24,7 +24,6 @@ message Request {
     RequestEcho               echo                 = 1;
     RequestFlush              flush                = 2;
     RequestInfo               info                 = 3;
-    RequestSetOption          set_option           = 4;
     RequestInitChain          init_chain           = 5;
```

Comment:
@melekes do we want to fix the number order here?

Reply:
v0.34 is breaking anyway, so I'd say yes